examples/component-versioning/component-mutable.md

```diff
@@ -95,7 +102,7 @@ spec:
   - env:
     - name: TEST_ENV
       value: test
-    image: wordpress:4.6.1-apache
+    image: wordpress:php7.2
```

Comment:
if we don't have parameter, then the change won't happen.

Reply:
i see, when finish the modification above (https://github.com/crossplane/oam-kubernetes-runtime/pull/142#discussion_r461350788) , i will restore this part.

Reply:
fixed